lib/instrumentation/openai.js

```diff
@@ -99,12 +99,18 @@ function addLlmMeta({ agent, segment }) {
  *
  * @param {object} params input params
  * @param {Agent} params.agent NR agent instance
+ * @param {Shim} params.shim the current shim instance
  * @param {TraceSegment} params.segment active segment from chat completion
  * @param {object} params.request chat completion params
  * @param {object} params.response chat completion response
  * @param {boolean} [params.err] err if it exists
  */
-function recordChatCompletionMessages({ agent, segment, request, response, err }) {
+function recordChatCompletionMessages({ agent, shim, segment, request, response, err }) {
```

Comment:
this file also has instrumentation to record embeddings. Right now it's still doing that